src/main/java/uk/gov/justice/digital/delius/controller/secure/AuthenticationController.java

```diff
@@ -59,6 +59,24 @@ public UserDetails findUser(@ApiParam(name = "username", value = "LDAP username"
                 .orElseThrow(() -> new NotFoundException(String.format("User with username %s", username)));
     }
 
+    @ApiOperation(
+            value = "Find user details of a user held in Delius Identity (LDAP)",
```

Comment:
minor, but worth amending the description

Reply:
fixed